packages/backend-tasks/src/tasks/PluginTaskSchedulerJanitor.ts

```diff
@@ -53,15 +53,31 @@ export class PluginTaskSchedulerJanitor {
 
   private async runOnce() {
     const dbNull = this.knex.raw('null');
+    let tasks = [];
+    const configClient = this.knex.client.config.client;
 
-    const tasks = await this.knex<DbTasksRow>(DB_TASKS_TABLE)
-      .where('current_run_expires_at', '<', this.knex.fn.now())
-      .update({
-        current_run_ticket: dbNull,
-        current_run_started_at: dbNull,
-        current_run_expires_at: dbNull,
-      })
-      .returning(['id']);
+    if (configClient.includes('sqlite3') || configClient.includes('mysql')) {
```

Comment:
Would you mind adding a test file for this class? You can take inspiration from the other test files nearby such as `TaskWorker.test.ts` where it creates some test databases to run against. If you have the `MYSQL_8` constant in the list, you know that the test is run against it and exercises this exact code